resources/views/models/blog/articles/index.blade.php

```diff
@@ -0,0 +1,56 @@
+@extends('lego::layouts.lego')
+
+@push('styles')
+    <link href="{{ asset('vendor/blog/css/blog.css') }}" rel="stylesheet">
+@endpush
+
+@section('content')
+    <x-fab::layouts.page
+        title="Blog Articles"
+        :breadcrumbs="[
+            ['title' => 'Home', 'url' => '/admin'],
```

Comment:
You can reference the dashboard with a route instead of hardcoding it `route('lego.dashboard')`.
You can do a search and replace for `'/admin'` with `route('lego.dashboard')` throughout all the files.